Source/PlasticSourceControl/Private/PlasticSourceControlOperations.cpp

```diff
@@ -192,42 +192,49 @@ bool FPlasticCheckInWorker::Execute(FPlasticSourceControlCommand& InCommand)
 		ParamCommitMsgFilename += FPaths::ConvertRelativePathToFull(CommitMsgFile.GetFilename());
 		ParamCommitMsgFilename += TEXT("\"");
 		Parameters.Add(ParamCommitMsgFilename);
-		// Detect special case for a partial checkout (CS:-1 in Gluon mode)!
-		if (-1 != InCommand.ChangesetNumber)
-		{
-			Parameters.Add(TEXT("--all")); // Also files Changed (not CheckedOut) and Moved/Deleted Locally
-		//  NOTE: --update added as #23 but removed as #32 because most assets are locked by the Unreal Editor
-		//	Parameters.Add(TEXT("--update")); // Processes the update-merge automatically if it eventually happens.
-			InCommand.bCommandSuccessful = PlasticSourceControlUtils::RunCommand(TEXT("checkin"), Parameters, InCommand.Files, InCommand.Concurrency, InCommand.InfoMessages, InCommand.ErrorMessages);
-		}
-		else
+		if (InCommand.Files.Num() > 0)
```

Comment:
There is a mismatch between the "if" and the "else" (their are not at the same indentation) so the new warning log will be called in the wrong situation
edit: just put the test at the beginning of the function, eg just before the log